src/main/java/ai/aliz/jarvis/context/ContextType.java

```diff
@@ -0,0 +1,10 @@
+package ai.aliz.jarvis.context;
+
+public enum ContextType {
+    BigQuery,
```

Comment:
you can add the set of the required parameters here and use them for validation later.

Reply:
Done.